### DISCUSSION TITLE
Final Rmarkdown File--No option to knit?

### DISCUSSION THREAD

**Discussion by shipmadison** [atsa-es/fish550-2023#22] [Q&A] [answered]
Hi all,

I've been troubleshooting an issue with our final Rmarkdown file. I've gotten it to render, but it hasn't created an HTML file, and I have no option to knit. There is a final version of our report in our file (Lab-2, Team-4) named Lab-2-team-4_WriteUp_Final, which has been pushed to Git. 

Can you help me identify the issue?

**Comment by mdscheuerell** [comment, marked as answer]
Hi @shipmadison. It looks like that sub-folder is somehow interacting with @eeholmes' Quarto doc in the main folder. Your **knit** button has likely been replaced with an arrow for rendering Quarto docs (see below).

When I render the doc it produces an `.html` doc, but Git is ignoring and so I can't push it to GitHub.

<img width="973" alt="Screen Shot 2023-04-26 at 10 46 57 AM" src="https://user-images.githubusercontent.com/5048528/234661101-7051adfa-8c07-4a36-8321-53f225d3db10.png">

**Comment by shipmadison** [comment]
Hi @mdscheuerell  

Thanks for the reply! I'll mess with the subdirectories to see if that helps. You are correct that I only saw the arrow render button. I did get it to produce an HTML document eventually using rmarkdown::render(), but it took a frustrating amount of troubleshooting to get there (I probably should have started with the console command). I'll be careful to mind my subdirectories in the future. Thanks again!

**Comment by mdscheuerell** [comment]
To clarify, @shipmadison, I don't think the issue has to do with _your_ directory structure per se, but rather the broader nested setup we have now.

**Comment by eeholmes** [comment]
@shipmadison. Mark is right. I messed up the knitting by adding a _quarto.yml file. Let me remove that.

**Comment by eeholmes** [comment]
@shipmadison Ok, do a pull from github and restart your project in RStudio. Just click the projects button in upper right and click on your project name (prob fish550?)
